CHANGELOG.md

```diff
@@ -58,6 +58,8 @@ Increment the:
   [#2124](https://github.com/open-telemetry/opentelemetry-cpp/pull/2124)
 * [API] Add user facing Logging API and Benchmarks
   [#2094](https://github.com/open-telemetry/opentelemetry-cpp/pull/2094)
+* [DEPRECATION] Drop C++11 support
+  [#2146](https://github.com/open-telemetry/opentelemetry-cpp/pull/2146)
```

Comment:
These lines are not added by this PR, and it should not show here.

There is something wrong the way merged from the main branch to this PR branch are done.